up-and-running/auth/custom.md

```diff
@@ -0,0 +1,92 @@
+# Custom Authentication
+
+If authentication system use in your organization is not listed in [npme's supported authentication methods](/up-and-running/auth/#authentication) you can use custom authentication, we have provided following two custom authentication plugins:
+ * npme-auth-atlassian-stash
+ * npme-auth-gitlab
+
+You can also opt to write your own `custom auth-plugin` for npm Enterprise.
```

Comment:
maybe add, `using one of the open-source plugins listed above as a starting point`.